docs/PacketFence_MSPKI_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,381 @@
+If you select 'MAC address', a certificate will be generated for the device itself using the MAC address as the identifier.
+If you select 'Username', a certificate will be generated for the user using his login name on the authentication backend (e.g. Active-Directory).
+
+This means that revoking the certificate for a username based certificate will block all the devices that this user registered.
+If you generate the certificates using the MAC address, revoking a certificate will block only that device.
+
+Provisioners configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Provisioners allow devices to automatically configure themselves to connect to the proper SSID (if applicable), use the proper authentication method (e.g. EAP-TLS) and trust the CA certificate and any certificate signed by it.
+
+Provisioners are configured in the PacketFence administration GUI under Configuration > Users > Provisioners.
+
+Add a new provisioner for each of the classes of devices to be supported amongst Android, Apple Devices and Windows.
+Fill out the form, choosing a different Provisioning Id per provisioner.
+
+The fields affect the provisioning behavior in the following fashion:
+
+* Roles: The "Roles" field defines which devices will be affected by the provisioning item. If empty all devices for this class will be affected.
+* SSID: The "SSID" field defines which SSID will be configured on the device using the authentication profile.
+* EAP-Type: The EAP type defines the authentication method supported and should be set to EAP-TLS to integrate with the PacketFence PKI.
+* Security type: The security type should be set to WPA2 to integrate with the PacketFence PKI.
+* PKI Provider: This should match the provider you configured earlier in the section on providers.
+
+The following is an example on how to configure an EAP-TLS connection for Windows/Android/OS X/iOS
+
+image::docs/images/scep-ms-pki-eaptls-example.png[scaledwidth="100%",alt="Provisioner EAP-TLS configuration"]
+
+OS X/iOS require signing the provisioning profile with a Certification Authority already trusted by the device such as e.g. VeriSign.
+Configuring this has to be done in the 'Signing' tab in the "Apple devices". 
+
+image::docs/images/packetfence-pki-eaptls-sign-example.png[scaledwidth="100%",alt="Signing provisioner"]
+
+Fill out the fields with the contents of the Base64 encoded certificates.
+To extract this information from a pem formatted certificate, copy the file content included between the begin and end tag, not including the delimiters themselves.
+
+Certificate file example:
+
+----
+----- BEGIN CERTIFICATE -----
+1234567890asdfghjkl
+zxcvbnmqwertyuiop78
+----- END CERTIFICATE -----
+----
+
+Copy everything between the BEGIN and END lines, but not the lines themselves.
+Repeat this operation for the certificate key and intermediate certificate if any.
```

Comment:
Why 'if any', they should be provided unless there will only be the CA and it will be useless for the signing